10up/wp-scrubber
# Added package-lock and gitignore; removed build files.

### Description of the Change

- Added `gitignore`
- Removed build files
- Added package-lock.json

### How to test the Change

- Install plugin via `git clone`
- Run build process: `npm install; composer install`
- Run `git status` and confirm there are no changes shows

### Changelog Entry

> Fixed - Bug fix

Resolved build files showing in git changes.

### Credits
Props @jamesmorrison


### Checklist:
